### PR TITLE
Smc 1359 Don't try and delete index that does not exist

### PR DESCRIPTION
**Pull request by ErasmusJW**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
+# 4.1.1 2020-11-04
+- Don't try to delete indexes that don't exist
```
Comment (JensEggers) [inline RELEASES.md]: please add ticket number into history

File: index.js [hunk 2]
```diff
@@ -22,10 +22,9 @@ function ElasticProvider(config) {
 
   if (!config.elasticCallConcurrency) config.elasticCallConcurrency = 20;
 
-  if (!config.bulkBachSizes) config.bulkBachSizes  = 1000;
-
-  if (!config.bulkMaxSize) config.bulkMaxSize  = 100000;
+  if (!config.bulkBachSizes) config.bulkBachSizes = 1000;
```
Comment (JensEggers) [inline index.js]: is it still possible to fix spelling of this field name? Should be bulkBatchSizes (with a t)

File: index.js [hunk 3]
```diff
@@ -291,16 +290,16 @@ function __bulk(path, setData, options, callback) {
...
   let bulkData = setData;
 
-
-
   // coming in from happn, not an object but a raw [] so assigned to data.value
   if (setData.data && setData.data.value) bulkData = setData.data.value;
   if (bulkData.length > _this.__config.bulkMaxSize)
-    throw new Error(`bulk batches can only be ${_this.__config.bulkMaxSize} entries or less ammount ${bulkData.length}`);
+    throw new Error(
+      `bulk batches can only be ${_this.__config.bulkMaxSize} entries or less ammount ${bulkData.length}`
```
Comment (JensEggers) [inline index.js]: typo - amount wants a single 'm'

**Review by JensEggers — APPROVED** [review]
minor comments only, pls fix what you can

File: index.js (hunk 3 above)
Comment (ErasmusJW) [reply index.js]: done

File: index.js (hunk 2 above)
Comment (ErasmusJW) [reply index.js]: done

File: RELEASES.md (hunk 1 above)
Comment (ErasmusJW) [reply RELEASES.md]: done

**Review by cordellcalitz — APPROVED** [review]

**Pull request merged**